cmd/convox/rack.go

```diff
@@ -205,6 +205,27 @@ func cmdRackParamsSet(c *cli.Context) error {
 		params[parts[0]] = parts[1]
 	}
 
+	existing, err := rackClient(c).ListParameters(system.Name)
+	if err != nil {
+		return stdcli.Error(err)
+	}
+
+	hasChange := false
+	for k, v := range params {
+		res, ok := existing[k]
+		if !ok {
+			hasChange = true
+			break
+		} else if res != v {
+			hasChange = true
+			break
+		}
+	}
+
+	if !hasChange {
```

Comment:
A parameter change is not the only thing that can cause a stack to change. Sometimes we update the underlying templates and the user will get changes even without parameter changes.

Let's not try to reimplement CloudFormation here. Are we just looking for a nicer display of their error?